frontend/src/pages/start/libs/components/industry-form/industry-form.tsx

```diff
@@ -1,3 +1,5 @@
+import { type RefObject } from 'react';
```

Comment:
let's avoid refs

Reply:
Then what type should I use?

Reply:
i think you can use this approach - https://www.w3schools.com/tags/att_button_form.asp